src/aleth/filter_api.cpp

```diff
@@ -9,23 +9,23 @@ using namespace eth;
 using namespace util;
 
 struct FilterAPIImpl : virtual FilterAPI {
-  optional<LogFilter> getLogFilter(FilterID id) const override { return std::nullopt; }
+  optional<LogFilter> getLogFilter(FilterID /*id*/) const override { return std::nullopt; }
```

Comment:
have you considered using __attribute__((unused)) ? Not sure what is better practice for this, just thinking out loud...

Reply:
attribute((unused)) works only for gcc/clang, but does not for MSVC.